frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -41,15 +49,17 @@ export interface IFormData {
   name: string;
   newUserType?: NewUserType | null;
   password?: string | null;
-  sso_enabled?: boolean;
+  new_password?: string | null; // if a new password is being set for an existing user, the API expects `new_password` rather than `password`
```

Comment:
It would be handy to rename this type `IUserFormData`, looks less ambiguous up the component tree